co-simulation/lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/detector/DetectedObject.java

```diff
@@ -37,15 +37,15 @@ public final class DetectedObject implements Serializable {
 
     private CartesianPoint position;
 
-    private Double[] positionCovariance = new Double[9];
+    private Double[][] positionCovariance = new Double[3][3];
```

Comment:
Just curious whether if there is any change in rest of the pipeline until this message reaches SDSM convertor in carma-street

Reply:
I do not believe so. V2X-Hub treats the JSON as a string and passes it on and the SensorDetectedObject has not been implemented on CARMA-Streets yet.